tests/e2e/stablehlo_ops/CMakeLists.txt

```diff
@@ -686,3 +686,183 @@ iree_check_single_backend_test_suite(
   COMPILER_FLAGS
     "--iree-input-type=stablehlo"
 )
+
+if(IREE_ROCM_TEST_TARGET_CHIP MATCHES "^gfx")
+
+unset(IREE_HIP_TEST_COMPILER_FLAGS)
+if(IREE_ROCM_TEST_LINK_BITCODE)
+  list(APPEND IREE_HIP_TEST_COMPILER_FLAGS "--iree-rocm-link-bc=true")
+else()
+  list(APPEND IREE_HIP_TEST_COMPILER_FLAGS "--iree-rocm-link-bc=false")
+endif()
+list(APPEND IREE_HIP_TEST_COMPILER_FLAGS
+  "--iree-rocm-target-chip=${IREE_ROCM_TEST_TARGET_CHIP}"
+  "--iree-rocm-bc-dir=${IREE_ROCM_TEST_BITCODE_DIR}")
```

Comment:
relating to other comments - not a fan of mixing rocm and hip flags here

Reply:
Simplified quite a lot here.